server.go

```diff
@@ -186,6 +235,248 @@ func main() {
 	}
 }
 
+const msgUpstreamParticipantCreateError = "could not create upstream participant. response status: %s"
+
+func upstreamNewParticipant(c echo.Context, p participant) (id string, err error) {
+	item := leaderboard_item{}
+	item.UserName = p.GitHubName
+	item.Slug = p.GitHubName
+	item.Score = 0
+	item.Archived = false
+	item.Draft = false
+
+	payload := leaderboard_payload{}
+	payload.Fields = item
+
+	body, err := json.Marshal(payload)
+	if err != nil {
+		return
+	}
+
+	req, err := http.NewRequest("POST", fmt.Sprintf("%s/collections/%s/items?live=true", webflowBaseAPI, webflowCollection), bytes.NewReader(body))
+	if err != nil {
+		return
+	}
+	req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", webflowToken))
+	req.Header.Add("Content-Type", "application/json")
+	req.Header.Add("accept-version", "1.0.0")
```

Comment:
You could likely strip this and the other use of it out into a func that modifies the request, or just returns the headers to add (I can't remember if Echo has a function for adding a slice of them, etc..., but it likely does?)

Reply:
consolidate header logic 92762cf